content/tutorials/spatial_wfs_services/index.md

```diff
@@ -422,14 +424,6 @@ The map of regions of Belgium.
 ``` r
 wfs_regions <- "https://eservices.minfin.fgov.be/arcgis/services/R2C/Regions/MapServer/WFSServer"
 url <- parse_url(wfs_regions)
-url$query <- list(service = "WFS",
-                  version = "2.0.0",
-                  request = "GetCapabilities")
-request <- build_url(url)
```

Comment:
OK to drop this (it was shown in the beginning), but perhaps not bad to show how you came to choosing the appropriate values for the GetFeature request in this example? Or, leave that as an exercise to the reader, but then do so explicitly.

Reply:
You are right. I have changed the code in 6e57c9c so it is a bit more didactic.